docs/tutorials/npm/src/ts/data-provider-manager.ts

```diff
@@ -0,0 +1,124 @@
+import {
+    ICU4XDataProvider,
+    ICU4XLocale,
+    ICU4XLocaleFallbacker,
+    ICU4XLocaleFallbackConfig,
+    ICU4XLocaleFallbackPriority
+} from 'icu4x';
+import * as localeDefault from '../../dist/locales';
+
+export class DataProviderManager {
+
+    private dataProvider: ICU4XDataProvider;
+    private fallbacker: ICU4XLocaleFallbacker;
+    private fallbackLocale: ICU4XLocale;
+    private loadedLocales: Set <ICU4XLocale> ;
+
+    private constructor() {
+        this.loadedLocales = new Set < ICU4XLocale > ();
+    }
+
+    public static async create(): Promise <DataProviderManager> {
+        const manager = new DataProviderManager();
+        await manager.init();
+        return manager;
+    }
+
+    private async init() {
+
+        const enFilePath = 'dist/en.postcard';
+        let enProvider = await this.createDataProviderFromBlob(enFilePath);
+        this.loadedLocales.add(ICU4XLocale.create_from_string("en"));
+
+        const unFilePath = 'dist/en.postcard';
+        let unProvider = await this.createDataProviderFromBlob(unFilePath);
+
+
+        let fallbacker = ICU4XLocaleFallbacker.create(unProvider);
+        let fallbackConfig = new ICU4XLocaleFallbackConfig();
+        fallbackConfig.priority = ICU4XLocaleFallbackPriority.Language;
+        let fallbackerWithConfig = fallbacker.for_config(fallbackConfig);
+
+        this.fallbacker = fallbackerWithConfig;
+        enProvider.enable_locale_fallback_with(this.fallbacker);
```

Comment:
Actually I wanted to display the `fallbacked` locale. `ICU4XLocaleFallbackerWithConfig` gives an iterator which helps to get the fallback, not sure how to achieve this using `ICU4XLocaleFallbacker`.

Also it doesn't through an error 😕

Reply:
The resolved locale is returned in `DataResponseMetdata.locale` (`None` if it's the request locale).

Reply:
Just to test the new fallbacker instead of `ICU4XLocaleFallbackerWithConfig`. I tried to write this function, but couldn't create the formatter:

```javascript

let enProvider = await createDataProviderFromBlob('dist/en.postcard');
let unProvider = await createDataProviderFromBlob('dist/und.postcard');
let fallbacker = ICU4XLocaleFallbacker.create(unProvider);
enProvider.enable_locale_fallback_with(fallbacker);
const locale = result(() => ICU4XLocale.create_from_string("bn"));
const groupingStrategy = ICU4XFixedDecimalGroupingStrategy.Auto;
const formatter = result(() => ICU4XFixedDecimalFormatter.create_with_grouping_strategy(
    enProvider,
    unwrap(locale),
    groupingStrategy
));

Reply:
(oh this is not Rust, ignore my comment).